src/main/scala/org/apache/spark/mllib/linalg/distributed/BetterBlockMatrix.scala

```diff
@@ -0,0 +1,188 @@
+package org.apache.spark.mllib.linalg.distributed
```

Comment:
I might have been using this when you commented it. I edited this file in fits and starts. Either way it's gone now.